### PR TITLE
Add documentation for PEM_{read,write}_bio_Parameters()

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: doc/man3/PEM_read_bio_PrivateKey.pod [hunk 1]
```diff
@@ -215,6 +219,12 @@ a DSA structure. The public key is encoded using a
 SubjectPublicKeyInfo structure and an error occurs if the public
 key is not DSA.
 
+The B<Parameters> functions read or write a key parameters in PEM format using
```
Comment (mattcaswell) [inline doc/man3/PEM_read_bio_PrivateKey.pod]: "read or write key parameters"

**Review by mattcaswell — CHANGES_REQUESTED** [review]
Minor typo

File: doc/man3/PEM_read_bio_PrivateKey.pod (hunk 1 above)
Comment (levitte) [reply doc/man3/PEM_read_bio_PrivateKey.pod]: Fixed

**Review by mattcaswell — APPROVED** [review]

**Comment by levitte** [comment]
Merged.

master:
9a6abb95be42b88c7c5ebc8c97f14afdc5919aa1 Add documentation for PEM_{read,write}_bio_Parameters()

1.1.1:
95e4ee81decfd7c832e499ca99b3e49886fe96c1 Add documentation for PEM_{read,write}_bio_Parameters()

**Pull request closed without merging**